isis/src/base/apps/camstats/camstats.cpp

```diff
@@ -0,0 +1,221 @@
+
+#include "camstats.h"
+#include "Camera.h"
+#include "CameraStatistics.h"
+#include "Cube.h"
+#include "Distance.h"
+#include "IString.h"
+#include "UserInterface.h"
+#include "Process.h"
+#include "Progress.h"
+#include "Statistics.h"
+
+
+using namespace std;
+using namespace Isis;
+
+namespace Isis {
+  //function to build stats data
+  void buildStats(Camera *cam, int &sample, int &line);
+  void writeFlat(ofstream &os, const Statistics *s);
+  string valueToString(const double &value);
+
+  void camstats(UserInterface &ui, Pvl *log) {
```

Comment:
I would write a quick docstring for this. Probably not worth doing the other since we plan to remove those functions.